tests/e2e/tests/generate/component/component-module.ts

```diff
@@ -0,0 +1,15 @@
+import {join} from 'path';
+import {ng} from '../../../utils/process';
+import {expectFileToMatch} from '../../../utils/fs';
+
+
+export default function() {
+  const modulePath = join('src', 'app', 'app.module.ts');
+
+  return ng('generate', 'component', 'test-component', '--module', 'app.module.ts')
+    .then(() => expectFileToMatch(modulePath,
+      /import { TestComponentComponent } from '.\/test-component\/test-component.component'/))
+
+    // Try to run the unit tests.
+    .then(() => ng('test', '--single-run'));
```

Comment:
This won't actually test if the import was correctly added, since we have no tests for `app.module.ts` and it's not included in the `test.ts` compilation. You can do `ng build` instead though, that'll compile it and thus verify the import.